news/2024/2024-05-21-the-perennial-returns.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: "The Perennial Returns"
+date: 2024-05-21 12:00:00 +0000
+---
+
+After a year-long hiatus we're brushing off the dust! The Perennial returns for its third iteration – and we've got more excitement than ever!
+
+![](/wiki/shared/news/2024-05-21-the-perennial-returns/Banner@2x.png)
+
+**[The Perennial](https://www.theperennial.net/)**, is a recurring open rank, international tournament using classic, polished formats and unique additions to bring excitement throughout the tournament. In 2021, the Perennial was one of very few international tournaments to ship trophy prizes across the world to the top three teams. In 2022, the Perennial was the first tournament in osu! to commission a song performed by a live band as an original track for the tournament. Further, the Perennial in 2022 became the second osu! Standard community tournament to ever receive the honor of being tri-badged, with the top three teams in the tournament all receiving profile badges for their accomplishment. This year promises to be better than either of our preceding iterations with more original tracks, competition just as tough, and some surprises along the way.
```

Comment:
very long. removed a little bit of fluff in the beginning

```suggestion
**[The Perennial](https://www.theperennial.net/)** is a recurring open rank, international tournament, notably being one one of very few international tournaments to ship trophy prizes across the world to the top three teams in 2021.

In 2022, the Perennial was the first tournament in osu! to commission a song performed by a live band as an original track for the tournament, as well as being the first osu! community tournament to be "tri-badged", with the top three teams in the tournament all receiving profile badges for their accomplishment.

This year promises to be better than both of the preceding iterations with more original tracks, competition just as tough, and some surprises along the way.
```

Reply:
Need to change "first osu! community tournament to be tri-badged" to second, we were not the first